packages/common/src/services/audius-backend/solana.ts

```diff
@@ -282,3 +312,94 @@ export const purchaseContent = async (
     await audiusBackendInstance.getAudiusLibs()
   ).solanaWeb3Manager!.purchaseContent(config)
 }
+
+export const findAssociatedTokenAddress = async (
+  audiusBackendInstance: AudiusBackend,
+  { solanaAddress, mint }: { solanaAddress: string; mint: MintName }
+) => {
+  return (
+    await audiusBackendInstance.getAudiusLibsTyped()
```

Comment:
Is `getAudiusLibsTyped` recent? Wasn't aware we had that.

Reply:
Not sure!